docs/docs/widgets/circular-progress-bar.md

```diff
@@ -0,0 +1,25 @@
+# Circular Progress Bar
+
+Circular progress bar widget can be used to show progress in a progress circle.
+
+<img class="screenshot-full" src="/img/widgets/circular-progressbar/circular-progress.gif" alt="ToolJet - Widget Reference - Tags" height="420"/>
+
+#### Properties
+
+| properties      | description |
+| ----------- | ----------- |
+| Text | We can set a text inside the progress circle.|
+| Progress | It can be used to set the progress of the widget. This will be a number and it must be between 0 to 100.|
```

Comment:
"This will be a number and" -> "Progress should be an integer between 0 and 100"